pkg/controller/migmigration/task.go

```diff
@@ -164,9 +161,6 @@ var CancelItinerary = Itinerary{
 		{phase: DeleteRestores},
 		{phase: EnsureStagePodsDeleted, all: HasStagePods},
```

Comment:
@aufi please note that once we fix Stage pod cleanup to run 1) during rollback, 2) at the beginning of migration, we should remove this stage pod removal step from the `CancelItinerary`.

So once you have a solution for https://github.com/konveyor/mig-controller/issues/700 we can strip out this phase here.